include/xmp.h

```diff
@@ -15,20 +15,26 @@ extern "C" {
 #define XMP_VER_MINOR 5
 #define XMP_VER_RELEASE 1
 
-#if defined(_WIN32) && !defined(__CYGWIN__)
+#if (defined(_WIN32) && !defined(__CYGWIN__)) || (defined(__OS2__) && defined(__WATCOMC__) && defined(__SW_BD))
```

Comment:
Please don't merge os/2 watcom case in here. It is wrong because dllimport isn't supported there and results in link failures.

Reply:
Oops, sorry about that. Fixed.